resources/js/mixins/ActionsColumn.js

```diff
@@ -53,7 +53,7 @@ export default {
      * @property actionColumnDefinition
      */
     actionColumnDefinition () {
-      return { key: 'actions', label: this.$t('app.actions'), sortable: false };
+      return { key: 'actions', label: this.$t('app.actions'), sortable: false, thClass: 'actionColumn' };
```

Comment:
I don't know if that is really a great idea. May be it is. But if there more than the standard amount of actions it must be possible to adjust the width for the table.

Reply:
I changed it to be a data attribute, that can be overwritten by the view